src/components/VmDetails/cards/NicsCard/style.css

```diff
@@ -25,10 +25,23 @@
 .ip4-container,
 .ip6-container {
   font-size: 10px;
-  padding: 0 10px;
+  padding: 0 5px;
+  flex: 1 0 33.3333%;
+  max-width: 100%;
+  min-width: 80px;
+  padding-bottom: 3px;
 }
 .ip6-container {
-  padding-left: 0;
+  flex: 2 0 66.6667%;
+}
+
+.ip {
```

Comment:
This is just to add padding to the bottom of each IP address field.

The `ip[46]-container` classes also now add the same bottom padding.  There wasn't any padding between NICs list items perviously.

Maybe a rule for all but the first IP block to have top padding?  This should allow for spacing between IP address blocks without padding added elsewhere....
````css
.ip + .ip {
  padding-top: 3px;
}
````

Reply:
@lwrigh thought it might look a little less clustered to have some padding between the IP addresses, especially since they're of very small font size